src/tools/fuzzing/fuzzing.cpp

```diff
@@ -3176,13 +3196,6 @@ Expression* TranslateToFuzzReader::makeRefCast(Type type) {
   return builder.makeRefCast(make(refType), type, RefCast::Safe);
 }
 
-Expression* TranslateToFuzzReader::makeI31New(Type type) {
```

Comment:
Was this never called?

Reply:
It was called by being taken by address in `make()`. But it is no longer needed, since now we add `make*Ref()` there, and `i31` is just one of the cases handled by those methods.